convert2rhel/backup.py

```diff
@@ -314,7 +317,19 @@ def restore(self):
             # IOError for py2 and OSError for py3
             loggerinst.warning("Error(%s): %s" % (err.errno, err.strerror))
             return
-        loggerinst.info("File %s restored." % self.filepath)
+
+        if rollback:
+            loggerinst.info("File %s restored." % self.filepath)
+        else:
+            loggerinst.debug("File %s restored." % self.filepath)
```

Comment:
This message here is not making too much sense at the moment, I would say... 

If it take as an example the same condition above, we are issuing an `loggerinst.task()` when we have a rollback, and a `loggerinst.info()` when we are not rollbacking. 

I would suggest following the same pattern as above, but this is not a blocker. 

```suggestion
        if rollback:
            loggerinst.task("Rollback: File %s restored." % self.filepath)
        else:
            loggerinst.info("File %s restored." % self.filepath)
```

Reply:
I'd disagree. If it's a rollback we'd want to have it as a step, so it would be 
```
[Task] Rollback: Restoring %s from backup
[Info] Rollback succeeded
```

Since at that step we care about the rollback and not about a generic backup we should make a not of it. If it's not a rollback it should be part of the logs as normal, as it's part of the overall task it's part of
```
[Info] Rollback: Restoring %s from backup
[Debug] Rollback succeeded
```

I wouldn't change it to two tasks as that separates it and creates a division that can be understood in the logs as being part of two functions

Reply:
The original non-task way is IMHO correct. We want just the first line to have the TASK prefix, and the "confirmation" message to be just of an info level below it. Example of what we have now:
```
10:34:02                 out: [11/29/2022 05:34:02] TASK - [Rollback: Restoring /etc/system-release from backup] **************
10:34:02                 out: File /etc/system-release restored.
```

With your proposal it would be:
```
10:34:02                 out: [11/29/2022 05:34:02] TASK - [Rollback: Restoring /etc/system-release from backup] **************
10:34:02                 out: [11/29/2022 05:34:02] TASK - [Rollback: File /etc/system-release restored.] **************
```

Reply:
Makes sense to me. Thanks for the explanation